Sources/PackageCollectionGenerator/PackageCollectionGenerate.swift

```diff
@@ -307,7 +307,9 @@ public struct PackageCollectionGenerate: ParsableCommand {
         //  - At most 3 minor versions per major version
         //  - Maximum of 2 majors
         //  - Maximum of 6 versions total
-        var allVersions = tags.compactMap { Version(string: $0) }
+        var allVersions = tags
+            .map{ $0.hasPrefix("v") ? String($0.dropFirst(1)) : $0 } // remove common "v" prefix which is supported by SwiftPM
+            .compactMap { Version(string: $0) }
```

Comment:
see https://github.com/mongodb/mongo-swift-driver/tags

Reply:
Run swiftformat?

Reply:
Does SwiftPM support "v" prefix by stripping it as well? Ideally `TSCUtility.Version` should support it.

Reply:
SwiftPM has this private extension:

```
extension Version {
    /// Try a version from a git tag.
    ///
    /// - Parameter tag: A version string possibly prepended with "v".
    init?(tag: String) {
        if tag.first == "v" {
            self.init(string: String(tag.dropFirst()))
        } else {
            self.init(string: tag)
        }
    }
}
```

Reply:
do you prefer I do that here too?

Reply:
> do you prefer I do that here too?

No it's ok. I just think the code should be shared somewhere--maybe make the extension in SwiftPM public, or add it to `TSCUtility.Version`, but then `TSCUtility.Version` implements semver and "v" prefix isn't part of the spec so the logic doesn't belong there.

Reply:
Anyway, it's something we can make nicer later. Doesn't have to be part of this PR.

Reply:
yea I agree this should go to TSC probably